### ISSUE TITLE
Gitlab link not working

### ISSUE THREAD

**Issue by ankittiwari101** [ArrjunPradeep/Medic-Chain#1]
[This]([https://gitlab.com/ced_b3_projects/ced-b3-g10.git]()) link mentioned in Readme file is not working.
More specifically, the page it is redirecting to on gitlab doesn't exist.
Please help as I am very much interested in using and studying this project for myself.

**Comment by mamonaabdullah122** [comment]
Yes same issue here yes i also very much interested in using and studying this project for myself.

**Comment by GuangguangLu** [comment]
This link mentioned in Readme file is not working.
